tests/distributed-transactions-test.py

```diff
@@ -64,11 +63,13 @@
                (pnodes, total_nodes-pnodes, topo, delay))
 
         Print("Stand up cluster")
-        extraNodeosArgs = ""
-        if speculative:
-           extraNodeosArgs = " --read-mode speculative "
+        specificExtraNodeosArgs = {}
+        specificExtraNodeosArgs[total_nodes-1] = f' --read-mode irreversible '
+        specificExtraNodeosArgs[total_nodes-2] = f' --read-mode irreversible '
+        specificExtraNodeosArgs[total_nodes-3] = f' --read-mode speculative '
+        specificExtraNodeosArgs[total_nodes-4] = f' --read-mode speculative '
```

Comment:
To reduce testing time and more clear, can they just be `specificExtraNodeosArgs[total_nodes-1] = f' --read-mode head '
        specificExtraNodeosArgs[total_nodes-2] = f' --read-mode irreversible '
        specificExtraNodeosArgs[total_nodes-3] = f' --read-mode speculative '

Reply:
https://github.com/AntelopeIO/spring/pull/1036/commits/72f673a775472bde42798d9bf460482019ee8743